docs/troubleshooting/detections/detection-rules.asciidoc

```diff
@@ -50,6 +50,14 @@ This can be caused by bad indicator data, a misconfigured rule, or too many even
 If you receive the following rule failure: `"An error occurred during rule execution: message: "Request Timeout after 90000ms"`, this indicates that the query phase is timing out. Try refining the time frame or dividing the data defined in the query into multiple rules.
 ====
 
+[discrete]
+[[IM-rule-heap-memory]]
+.Indicator match rules are failing because the `maxClauseCount` limit is too low.
+[%collapsible]
+====
+If you receive the following rule failure: `Bulk Indexing of signals failed: index: ".index-name" reason: "maxClauseCount is set to 1024" type: "too_many_clauses"`, this indicates that the limit for the total number of clauses that a query tree can have is too low. To update your maximum clause count, {ref}/advanced-configuration.html#set-jvm-heap-size[increase the size of your {es} JVM heap memory]. 1GB of {es} JVM heap size or more is sufficient.
```

Comment:
I noticed that the linked page in ES docs doesn't use a space for measurements, but I think including a space is much more standard in general. 
```suggestion
If you receive the following rule failure: `Bulk Indexing of signals failed: index: ".index-name" reason: "maxClauseCount is set to 1024" type: "too_many_clauses"`, this indicates that the limit for the total number of clauses that a query tree can have is too low. To update your maximum clause count, {ref}/advanced-configuration.html#set-jvm-heap-size[increase the size of your {es} JVM heap memory]. 1 GB of {es} JVM heap size or more is sufficient.
```

Reply:
Either work IMO and I've seen both in our solution docs. :)